wasm_bindgen/raze/BUILD

```diff
@@ -1,4 +1,5 @@
 """
```

Comment:
delete

Reply:
I actually meant to keep the `BUILD` file to reduce the amount of deltas in this PR. I've removed `BUILD.bazel` for now. This can be updated by removing `output_buildfile_suffix = "BUILD"` from the `Cargo.toml` file and rerunning `cargo-raze`